src/types/chats/requestquery.ts

```diff
@@ -0,0 +1,25 @@
+import { Message } from "../messages/message";
+
+export enum Status {
+	ONGOING = "ongoing",
+	AUTOREPLY = "autoreply",
+	PENDING = "pending",
+	CLOSED = "closed",
+	REVIEWED = "reviewed"
+}
+
+export enum Type {
+	UNKNOWN = "unknown",
+	QUERY = "query",
+	REQUEST = "request"
+}
+
+export type RequestQuery = {
```

Comment:
just curious: no strong opinion on this yet but
why types over interfaces here? :0

Reply:
I would back with Khang Hou here. Type >> interface unless you want to define methods for classes

https://www.youtube.com/watch?v=Idf0zh9f3qQ